scripts/globals/items/tube_of_healing_salve_ii.lua

```diff
@@ -0,0 +1,33 @@
+-----------------------------------------
+-- ID: 5836
+-- Item: Tube of Healing Salve II
+-- Item Effect: Restores 100% HP to Pet
+-----------------------------------------
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+    local result = 0
+    if not target:hasPet() then
+        result = dsp.msg.basic.REQUIRES_A_PET
+    end
+    if target:hasStatusEffect(dsp.effect.MEDICINE) then
```

Comment:
this is especially egregious since you aren't even the one effected by the "medicine" here. Your pet eats a pill but you get the buzz? Dayum that is some strong stuff! Who your dealer bruh?

Reply:
Contact high. That or he was shot gunning

Reply:
No more checks for medicine